packages/flutter_tools/test/integration.shard/flutter_build_windows_test.dart

```diff
@@ -0,0 +1,117 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file/file.dart';
+import 'package:file_testing/file_testing.dart';
+import 'package:flutter_tools/src/base/io.dart';
+
+import '../src/common.dart';
+import 'test_utils.dart';
+
+void main() {
+  // Only run this test on Windows
+  if (!platform.isWindows) {
+    return;
+  }
```

Comment:
Typically the way we do this is:

```dart
import 'dart:io' as io;
...
test('windows-specific test of some sort', () {
  // Insert exciting and meaningful test code here.
}, skip: !io.Platform.isWindows);
```

I believe `group()` also supports a skip predicate, which might make sense here since you'll want to skip everything. 

Using the raw dart:io Platform is handy for the `skip` since the tool `platform` global is dependency-injected and can be faked out, even though typically that happens explicitly on the test so the chance of it being anything other than the host OS here is basically zero.

Reply:
Ah both those are good to know! I copied this snippet from some other test, which I would've liked to fix as well but I'm unable to find it now. I fixed this test though!